sf_examples/megaverse_examples/experiments/megaverse_envs.py

```diff
@@ -0,0 +1,25 @@
+from sample_factory.runner.run_description import Experiment, ParamGrid, RunDescription
+
+_params = ParamGrid(
+    [
+        ("seed", [00, 11, 22, 33]),
+        (
+            "env",
+            [
+                "megaverse_TowerBuilding",
```

Comment:
Since the experiment is called "megaverse_envs", can we add all other envs here in the "env" list?
But maybe comment them out. This way it would be easy for ppl to understand how to run with other envs (just uncomment them).